rasa/importers/multi_project.py

```diff
@@ -162,13 +164,39 @@ def _is_in_imported_paths(self, path) -> bool:
 
     def add_import(self, path: Text) -> None:
         self._imports.append(path)
-
+    
+    def _log_merge_infos(self, domains) -> None:
+        merges = {
+            "actions": {},
+            "entities": {},
+            "forms": {},
+            "intents": {},
+            "slots": {}
+        }
+        domain_dicts = [domain.as_dict() for domain in domains]
```

Comment:
I'm personally not a huge fan of Hungarian notation. The type should be clear in my opinion. What do you think?
```suggestion
        domains = [domain.as_dict() for domain in domains]
```

Reply:
Also not a fan of Hungarian notation in general, but here I feel it might be confusing to call it just `domains` since this variable already exists in this scope with a different content (namely domains, instead of domain dictionaries).

If you have a better name in mind (or if you still prefer to stick to `domains`), let me know and I will modify it accordingly.